### ISSUE TITLE
[Kernel] Missing kernel module for Network (Wifi and LAN via USB dongle)

### ISSUE THREAD

**Issue by Nivalux** [serpent-os/recipes#246]
Hi there,
I tried installing the Pre Alpha ISO on my older Microsoft Surface and I cannot connect it to the network via wifi.
It's missing the kernel module.

I used the another Live ISO to see more information and here it is:

lsusb
Bus 001 Device 005: ID 1286:204b Marvell Semiconductor, Inc. Bluetooth and Wireless LAN Composite Device

lspci
01:00.0 Ethernet controller: Marvell Technology Group Ltd. 88W8897 [AVASTAR] 802.11ac Wireless

lspci -v
...
Kernel driver in use: mwifiex_pcie
Kernel modules: mwifiex_pcie

I tried using 
modprobe mwifiex_pcie
on the pre alpha ISO and it could not find the kernel module.

**Comment by Nivalux** [comment]
Update: I now tried it once more with an USB Hub + USB LAN Adapter and it also seems to be missing kernel modules (I think. Cannot check because no USB utils)
dmesg showed, that it did detect the USB hub and the LAN Adapter. But it did not load it as a network interface (as far as I can tell with dmesg).

With another live ISO it worked and I checked with the "usb-devices" command, which drivers are in use.
For my Surface Tablet it uses:
- hub
- r8152
- usbhid
- btusb
- uvcvideo
- usb-storage

The "r8152" is for the LAN Adapter. Not sure if the "hub" driver is also missing, but it probably should be included as well.
